xml/schema/logixng/digital-actions/display-action-positionable-by-class-5.5.4.xsd

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-stylesheet href="../schema2xhtml.xsl" type="text/xsl"?>
+
+<!-- This schema is part of JMRI. Copyright 2018.                           -->
+<!--                                                                        -->
+<!-- JMRI is free software; you can redistribute it and/or modify it under  -->
+<!-- the terms of version 2 of the GNU General Public License as published  -->
+<!-- by the Free Software Foundation. See the "COPYING" file for a copy     -->
+<!-- of this license.                                                       -->
+<!--                                                                        -->
+<!-- JMRI is distributed in the hope that it will be useful, but WITHOUT    -->
+<!-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or  -->
+<!-- FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License  -->
+<!-- for more details.                                                      -->
+
+<!-- This file contains definitions for LogixNG                             -->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:xsi ="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns:docbook="http://docbook.org/ns/docbook"
+           xmlns:jmri="http://jmri.org/xml/schema/JMRIschema"
+           xsi:schemaLocation="
+                http://jmri.org/xml/schema/JMRIschema http://jmri.org/xml/schema/JMRIschema.xsd
+                http://docbook.org/ns/docbook http://jmri.org/xml/schema/docbook/docbook.xsd
+            "
+        >
+
+    <xs:complexType name="LogixNG_DigitalAction_DisplayActionPositionableByClassType">
+      <xs:annotation>
+        <xs:documentation>
+          Define the XML stucture for storing the contents of a ActionTurnout class.
+        </xs:documentation>
+        <xs:appinfo>
+            <jmri:usingclass configurexml="true">jmri.jmrit.logixng.digital.actions.configurexml.ActionTurnoutXml</jmri:usingclass>
+        </xs:appinfo>
+      </xs:annotation>
+        
+            <xs:sequence>
+
+              <xs:element name="systemName" type="systemNameType" minOccurs="1" maxOccurs="1"/>
+              <xs:element name="userName" type="userNameType" minOccurs="0" maxOccurs="1"/>
+              <xs:element name="comment" type="commentType" minOccurs="0" maxOccurs="unbounded"/>
+              <xs:element name="editorName" type="xs:string" minOccurs="0" maxOccurs="1"/>
+              <xs:element name="className" type="xs:string" minOccurs="0" maxOccurs="1"/>
```

Comment:
No change needed, just an observation.

Should this and the related references in other areas be "classNames" since it is a comma separated list names.

Reply:
`className` is used when it's only a single class, which it is here.
`classNames` is used when it's multiple classes, for example when a Positionable is stored in the xml file.

The LogixNG action can only control items of one class.

But when the Positionable is edited, the user can enter several classes in a comma separated list.

Reply:
> `className` is used when it's only a single class, which it is here. `classNames` is used when it's multiple classes, for example when a Positionable is stored in the xml file.
> 
> The LogixNG action can only control items of one class.
> 
> But when the Positionable is edited, the user can enter several classes in a comma separated list.

You are correct.  I lost track of positionable label and LogixNG action xml requirements.